lib/matplotlib/collections.py

```diff
@@ -1404,11 +1404,12 @@ def __init__(self, segments,  # Can be None.
         Parameters
         ----------
         segments : list of array-like
-            A sequence of (*line0*, *line1*, *line2*), where::
+            A sequence (*line0*, *line1*, *line2*) of lines, where each line is a list
+            of points::
 
-                linen = (x0, y0), (x1, y1), ... (xm, ym)
+                lineN = [(x0, y0), (x1, y1), ... (xm, ym)]
 
-            or the equivalent numpy array with two columns. Each line
+            or the equivalent Mx2 numpy array with two columns. Each line
```

Comment:
(M, 2)

Reply:
I don't really care. This is a very quick PR to improve a misunderstanding in #26676. I have no time to bike shed this further. Feel free to push to this PR or take over if you think it's possible/worth improving further.